runner_manager/main.py

```diff
@@ -21,6 +25,13 @@ def startup_event():
     log.info(f"Startup job is {status}")
 
 
-@app.get("/_health")
-def health():
-    return Response(status_code=200)
+@app.get("/public")
+def public():
+    """A public endpoint that does not require any authentication."""
+    return "Public Endpoint"
+
+
+@app.get("/private")
+def private(api_key: str = Security(get_api_key)):
+    """A private endpoint that requires a valid API key to be provided."""
+    return f"Private Endpoint. API Key: {api_key}"
```

Comment:
We have no use for those for now, but they are usefull for testing, would you mind trying to set them as a router inside `test_auth.py` (check `routers/_health.py` for inspiration)

Then in your test include `fastapp` `def test_my_test(fastapp, ...):`

Then proceed in adding those routes for the tests only: `fastapp.include_router(my_auth_test_router)` then execute the tests.